### ISSUE TITLE
(deep fill v2)the mean L2 distance between predicted image and ground truth decreased and then increased

### ISSUE THREAD

**Issue by tangeroo** [JiahuiYu/generative_inpainting#339]
@JiahuiYu Hi,
  I trained this model with my own data and downsampled images were input.Before iters 10200 the mean L2 distance between predicted image and ground truth kept going down to 0.01(calculated by (-1,1)data,wasn't normalized to(0,255)), and subsequently which was up to 0.5 and even worse.The iters 10200 results are great,but I want higher accuracy actually.So do you have any suggestion?I really appreciate it.
Thank you.
![image](https://user-images.githubusercontent.com/25971461/68280966-93f8f800-00b1-11ea-8040-211bc0863fbd.png)
![image](https://user-images.githubusercontent.com/25971461/68280915-7c217400-00b1-11ea-8b36-5964fb6f02f0.png)
![image](https://user-images.githubusercontent.com/25971461/68280854-5c8a4b80-00b1-11ea-9330-6fc7d43aac71.png)

**Comment by JiahuiYu** [comment]
@tangeroo Hi thanks for your interest!

It seems like in tier 136000, the model failed? Can you try to train the model again from scratch? Are you using default hyper-parameter settings and datasets (Places2, CelebA-HQ, etc)?

**Comment by tangeroo** [comment]
@JiahuiYu Hi thanks for your reply
I did use default hyper-parameter settings but my own dataset not Places2 or celeA-HQ,etc.The best accuracy is 0.01 mentioned above in this situation, I wonder whether this model has the potential or not, that  I may make some changes to meet higher accuracy like 0.001 or 0.0001.
